src/effects/VST3/VST3Utils.h

```diff
@@ -14,6 +14,10 @@
 
 #include <string>
 #include <pluginterfaces/vst/vsttypes.h>
+#include <pluginterfaces/vst/ivstaudioprocessor.h>
```

Comment:
The addition of this `include` of this header allows the deletion of the other two.

Reply:
hmm I could not get rid of the #include <string> though